templates/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -3,12 +3,12 @@
 package {{invokerPackage}};
 
 {{#dynamicOperations}}
-import io.swagger.v3.oas.models.OpenAPI;
-import io.swagger.v3.oas.models.Operation;
-import io.swagger.v3.oas.models.PathItem;
-import io.swagger.v3.oas.models.parameters.Parameter;
-import io.swagger.v3.oas.models.parameters.Parameter.StyleEnum;
-import io.swagger.v3.parser.OpenAPIV3Parser;
+    import io.swagger.v3.oas.models.OpenAPI;
```

Comment:
There's too much changing just because of whitespace modifications. I don't know which lines you've changed because there are 3,375 lines changed and most of them are whitespace. Your PRs need to just include whichever details you've actually changed. There shouldn't be whitespace modifications like this.

Cannot review until the PR is limited to only the lines that you've changed.

Reply:
I've came back to previous formatting so you have changes what I needed only :)